JQuery/Custom-JQuery.js

```diff
@@ -0,0 +1,272 @@
+                });
+                break;
+
+            case 2:
+                const attributeValue = getStringOrFunctionValue(params[1]);
+
+                this.each((element) => {
+                    element.setAttribute(attributeName, attributeValue);
+                });
+        }
+    }
+
+    JQuery.prototype.children = function(...params) {
+        switch (params.length) {
+            case 0:
+                const children = [];
+
+                this.each((element) => {
+                    const childrenNodes = element.children;
+
+                    children.push([...childrenNodes]);
+                });
+                return children;
+            case 1:
+                const childrenArray = [];
+
+                this.each((element) => {
+
+                    const childrenElements = element.querySelectorAll(params[0]);
+
+                    for (const child of childrenElements)
+                        childrenArray.push(child);
+                });
+                return childrenArray;
+        }
+    }
+
+    JQuery.prototype.css = function(...params) {
+        switch (params.length) {
+            case 1:
+                const element=this.elements[0];
+                const firstParameter = params[0];
+                const properties = [];
+
+                if (!isString(firstParameter) && !isObject(firstParameter)) {
+                    return;
+                }
+                if(isString(firstParameter))
+                {
+                    const styleName=firstParameter;
+                    return getStyleValue(element, styleName);
+                }
+
+                if (Array.isArray(firstParameter)) {
+                    const styleNames=firstParameter;
+                    return getStylesValues(element, styleNames);
+                }
+                else {
+                    const stylesDictionary = firstParameter;
+                    addStyles(element, stylesDictionary);
+                }
+                break;
+
+            case 2:
+                const styleName = params[0];
+                const styleValue = getStringOrFunctionValue(params[1]);
+
+                addCssStyle(element, styleName, styleValue);
+        }
+    }
+
+    JQuery.prototype.empty = function() {
+        this.each((element) => {
+            while (element.hasChildNodes()) {
+                element.removeChild(element.childNodes[0]);
+            }
+        })
+    }
+
+    JQuery.prototype.wrap = function(elementTag) {
+        const wrappingElementTag = getStringOrFunctionValue(elementTag);
+
+        this.each((element) => {
+            appendAndWrapElement(element, wrappingElementTag);
+        });
+    }
+
+    return initialize;
+
+})();
+
+function addStyles(element, stylesDictionary) {
+    for (let key in stylesDictionary) {
+        addCssStyle(element, key, stylesDictionary[key]);
+    }
+}
+
+function getStylesValues(element, styleNames) {
+    let stylesValues=[];
```

Comment:
Please, check all the usages of `let` in your project and replace them with `const` where it is possible.